GOVERNANCE.md

```diff
@@ -0,0 +1,299 @@
+  - [Emeritus Maintainers](#emeritus-maintainers)
+- [Technical Advisory Groups](#technical-advisory-groups)
+- [Decision making](#decision-making)
+  - [Governance changes](#governance-changes)
+    - [Editorial changes](#editorial-changes)
+  - [Maintainership](#maintainership)
+  - [Sensitive decisions](#sensitive-decisions)
+  - [Ordinary decisions](#ordinary-decisions)
+- [Voting](#voting)
+  - [Consensus](#consensus)
+  - [Majority vote](#majority-vote)
+  - [Supermajority vote](#supermajority-vote)
+- [License](#license)
+
+## Principles
+
+The Falco Project and its community adhere to the following principles:
+ - **Open**: Falco is open source and open to contribution, accessible and welcoming for everyone.
+ - **Respectful**: the community pledges to respect all people involved in the project.
+ - **Transparent**: discussions, collaboration, and decision-making are done in public.
+ - **Vibrant**: evolution is better than stagnation.
+
+The Falco Project is part of the broader [CNCF](https://www.cncf.io/) community, and adheres to its values.
+
+## Project Evolution
+
+The project source code and documentation live in various repositories under the [falcosecurity](https://github.com/falcosecurity) GitHub organization. Each repository represents a component or a subproject of The Falco Project. For transparency, project decisions are publicly made using GitHub issues and pull requests.
+
+The Falco Project documents its evolution in the [falcosecurity/evolution](https://github.com/falcosecurity/evolution) repository, which is also a place designed to:
+ - make decisions that regard the whole project, and
+ - define rules and structures which span beyond the extent of a single repository (i.e., organization-wide).
+
+
+## Repositories
+
+The process of adding, removing, and managing repositories under the [falcosecurity](https://github.com/falcosecurity) GitHub organization is described in [REPOSITORIES.md](REPOSITORIES.md). The document assigns a scope and a status to each repository.
+
+### Core repositories
+
+In particular, repositories essential for building, installing, running, documenting, or using Falco are considered **core repositories** and are given the *Official* status.
+
+### Repository ownership
+
+[OWNERS](https://www.kubernetes.dev/docs/guide/owners/) files are used to designate responsibility over different parts of The Falco Project codebase, and serve as the implementation mechanism for the code review process. Each repository must have an `OWNERS` file in the root directory, and can optionally have other `OWNERS` files in subdirectories. The `OWNERS` file applies to everything within the directory, including the OWNERS file itself, sibling files, and child directories.
+
+## Community
+
+The Falco Project is driven by its community. The [falcosecurity/community](https://github.com/falcosecurity/community) repository documents the official communication channels, community calls, and other initiatives. 
```

Comment:
Meta information -- that is, information about the project overall unattached to specific tools -- seems to be spread around the /evolution, /community, and /falco repositories.  This is going to be very confusing for new contributors, and can easily lead to conflicting information between them.  Consider consolidating this kind of information in one place.